File: src/main/java/com/gmail/borlandlp/minigamesdtools/arena/team/lobby/starter/StarterLobby.java
package com.gmail.borlandlp.minigamesdtools.arena.team.lobby.starter;

import com.gmail.borlandlp.minigamesdtools.arena.ArenaPhaseComponent;
import org.bukkit.entity.Player;

import java.util.Set;

public interface StarterLobby extends ArenaPhaseComponent {
    void setEnabled(boolean trueOrFalse);
    void addPlayer(Player player);
    void removePlayer(Player player);
    Set<Player> getPlayers();
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/arena/team/lobby/starter/StarterLobby.kt
package com.gmail.borlandlp.minigamesdtools.arena.team.lobby.starter

import com.gmail.borlandlp.minigamesdtools.arena.ArenaPhaseComponent
import org.bukkit.entity.Player

interface StarterLobby : ArenaPhaseComponent {
    var isEnabled: Boolean
    fun addPlayer(player: Player)
    fun removePlayer(player: Player)
    val players: Set<Player>
}